bait-files/src/main/java/com/google/sps/servlets/UserHistoryServlet.java

```diff
@@ -26,6 +26,12 @@
 
 @WebServlet("/userHistory")
 public class UserHistoryServlet extends HttpServlet {
+  class SortByEarliestDate implements Comparator<Request> {
```

Comment:
This doesn't agree with changes made elsewhere.. specifically your other PR and andrew's user perms PR. Hope you can sync this to be consistent.